deploy/nl/embeddings.yaml

```diff
@@ -1,4 +1,4 @@
 small: embeddings_small_2023_05_24_23_17_03.csv
-medium_ft: embeddings_medium_2023_09_05_17_45_10.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+medium_ft: embeddings_medium_2023_09_17_17_22_04.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
```

Comment:
Could you try "families on food stamps", "families with SNAP benefits"?

Reply:
Yup, here: 
https://screenshot.googleplex.com/r8S9b7ZHpQQDs87 (families on food stamps)
https://screenshot.googleplex.com/AmjqCLbNMm2UE6j (families with SNAP benefits)

Reply:
Great!